src/api/app/views/webui/project/_project_packages.html.haml

```diff
@@ -11,11 +11,12 @@
     = link_to project.scmsync do
       This project is managed in SCM
   - elsif packages.present?
-    .mb-2{ data: { label: '' } }
-      Filters:
-      %span.d-none#label-filter
-        %span.badge
-      = link_to('Clear filters', '#', id: 'label-clear', class: 'label-filter btn btn-sm btn-primary')
+    - if Flipper.enabled?(:labels, User.session)
```

Comment:
This change looks good. :+1: 

However, later on, the javascript to filter by labels is using other feature flag. See: https://github.com/openSUSE/open-build-service/blame/5d8d2f0f51fcd4110d3c9176b25751847e55e24d/src/api/app/views/webui/project/_project_packages.html.haml#L63-L65

Could you please double check this is working as intended?